lib/davinci_dtr_test_kit/client_groups/light_ehr/dtr_light_ehr_user_response_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../urls'
+require_relative '../../tags'
+
+module DaVinciDTRTestKit
+  # noinspection RubyJumpError,RubyResolve
+  class DTRLightEHRUserResponseTest < Inferno::Test
+    include URLs
+    id :dtr_light_ehr_user_response
+    title 'Checks for a valid user response at the supported payer endpoint'
```

Comment:
We have a convention to put `[USER INPUT VERIFICATION]` at the beginning of the title of these tests that check user input for simulation purposes. Please add